en/customization/customization.md

```diff
@@ -1,7 +1,7 @@
 # Customization
 
 * [Introduction](introduction.md)
-* [Texts & Translations](translations.md)
+* [Translations & Texts](translations.md)
```

Comment:
Since we're replacing `&` with `y` in Spanish, my vote goes for replacing `&` with `and` in English as well. Using `&` is too informal for documentation IMHO.